### PR TITLE
Struggles at the Beginning – Software Testers journal

### PR DESCRIPTION
**Pull request by TehreemFatima07**

### CODE REVIEW

**Review by teodorovicmilan — APPROVED** [review]

**Review by markehammons — CHANGES_REQUESTED** [review]
More review to follow

File: posts/2022-11-07-software-qa-blog.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Communication is the key when it comes to setting the testing strategies for the project. It is useful because the quality of work improves as the quality of communication within the team increases. Get the feedback from developers and the product owner about the project. One can start by asking a simple set of questions like:
+
+- What is the development process of the project?
+- What are the users critical requirements?
+- What is working well in the project?
+- What are the most common problem cases?
+
+Once you get the answers to these basic questions, start observing how the things are actually working and write them down.
+
+Next step is to focus on setting the testing strategy for the project and check how the testing culture can be embedded in the team.
```
[the remark this answers is not in the document]
Comment (TehreemFatima07) [reply posts/2022-11-07-software-qa-blog.adoc:32]: Testing culture in general

[the remark this answers is not in the document]
Comment (markehammons) [reply posts/2022-11-07-software-qa-blog.adoc:32]: Remove "the" from testing culture then. On the opposite end, "Next step" is a specific step, so put "The" in front.

[the remark this answers is not in the document]
Comment (TehreemFatima07) [reply posts/2022-11-07-software-qa-blog.adoc]: Updated

**Review by markehammons — APPROVED** [review]

**Pull request merged**